src/QuadVelocityController.cpp

```diff
@@ -142,36 +160,53 @@ bool QuadVelocityController::update(const ros::Time& time,
     }
 
     // Print the velocity and throttle information
-    ROS_INFO("Vz: %f Vx: %f Vy: %f Vyaw: %f",
-             velocity.linear.z,
-             velocity.linear.x,
-             velocity.linear.y,
-             velocity.angular.z);
+    ROS_INFO("Vz: %f Vx: %f Vy: %f",
+             velocity.z,
+             velocity.x,
+             velocity.y);
     ROS_INFO("Throttle: %f Pitch: %f Roll: %f Yaw: %f",
              uav_command.throttle,
              uav_command.data.pitch,
              uav_command.data.roll,
              uav_command.data.yaw);
 
+    last_update_time_ = time;
     return true;
 }
 
 bool QuadVelocityController::waitUntilReady()
 {
-    geometry_msgs::TransformStamped transform;
-    bool success = getTransformAtTime(
-                       transform,
-                       ros::Time(0),
-                       ros::Duration(INITIAL_TRANSFORM_WAIT_SECONDS));
+    const ros::Time start_time = ros::Time::now();
+    while (ros::ok()
+           && odometry_msg_queue_.empty()
+           && ros::Time::now() < start_time + max_initial_wait_) {
+        ros::spinOnce();
+        ros::Duration(0.001).sleep();
```

Comment:
On line 212 we use a ros::Duration(0.005).sleep() I don't really care which we use. (I would prefer 0.005 though) but I think it should be consistent.